qiita_pet/support_files/portal.txt

```diff
@@ -0,0 +1,45 @@
+[sitebase]
```

Comment:
Rename file to `qiita_portal.txt` and add a comment on top as it is done for `qiita_core/support_files/config_test.txt`.